docs/03-07-environments-cleanup.md

```diff
@@ -1,13 +1,15 @@
-# Environments cleanup
+# Environments Cleanup CronJob
 
-Environments Cleanup is an application that cleans up environments that do not meet requirements in a given Gardener project.
+>**CAUTION:** The CronJob must run only in the development environment. You must be connected to the development Kubernetes cluster before applying the CronJob.
+
+Environments Cleanup CronJob removes Kyma Environments which are older than 24h. The CronJob is scheduled to run daily at midnight local time defined in the system.
 
 ## Prerequisites
 
 Environments Cleanup requires access to:
 - Gardener project of choice to filter Shoots without a proper label
-- Database to get an Instance ID for each Runtime marked for deletion
-- Kyma Environment Broker to trigger Runtime deprovisioning
+- the KEB database to get an Instance ID for each SAP BTP, Kyma runtime marked for deletion
```

Comment:
```suggestion
- the Kyma Environment Broker (KEB) database to get an Instance ID for each SAP BTP, Kyma runtime marked for deletion
```
Is this KEB something else than the following one? If yes, please discard this ant the following suggestion.